### PR TITLE
Adds programming experience to pre-filled survey link

### LINKED ISSUE #246: Online Survey is missing the programming experience
In the survey, the user programming experience is not pre-filled, even though this is known from the previous WatchDog user registration. Include a parameter to pre-fill this, too.

All other pre-filled fields work flawlessly, see attached screenshot.

screenshot from 2016-04-20 11-07-13

### PR DESCRIPTION
**Pull request by nspruit**
Fixes #246

### CODE REVIEW

File: WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/ui/util/UIUtils.java [hunk 1]
```diff
@@ -345,8 +347,15 @@ public static void createOpenReportLink(JComponent parent) {
      * the correct User ID.
      */
     public static void createStartDebugSurveyLink(JComponent parent) {
```
Comment (Inventitech) [inline WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/ui/util/UIUtils.java]: Couldn't this be in Core?

File: WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/ui/util/UIUtils.java [hunk 2]
```diff
@@ -345,8 +347,15 @@ public static void createOpenReportLink(JComponent parent) {
...
+        try {
+            programmingExperienceParam += URLEncoder.encode(Preferences.getInstance().getProgrammingExperience(), "UTF-8");
+        } catch (UnsupportedEncodingException e) {
+            programmingExperienceParam = "";
+        }
+
         String surveyLink = "https://docs.google.com/forms/d/1ybD1jC-iICXNlmQpyPEFngtmOtodicDr18E1ZbfBtx4/viewform?entry.1872114938="
-                + Preferences.getInstance().getUserId()
+                + Preferences.getInstance().getUserId() + programmingExperienceParam
                 + "&entry.87074017=Java&entry.1002919343=IntelliJ&entry.2010347695&entry.2084367812";
```
Comment (Inventitech) [inline WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/ui/util/UIUtils.java]: I think there might be a dynamic way to get the string "Eclipse" or "IntelliJ" (and the brutal way would be to include this in a Plugin-specific class, which would be fine by me) :-)

**Comment by nspruit** [comment]
Now, the process of building up the correct link is done inside the core project

**Comment by Inventitech** [comment]
OK, had to re-trigger a build (first test execution mysteriously failed), but now they're working! Well done.

**Pull request merged**